src/ofxstatement/plugins/iso20022.py

```diff
@@ -81,13 +87,17 @@ def _get_statement_el(self, tree: ET.ElementTree) -> ET.Element:
     def _parse_statement_properties(self, tree: ET.ElementTree) -> None:
         stmt = self._get_statement_el(tree)
 
-        bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:BIC", self.xmlns)
+        bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:BICFI", self.xmlns)  # checked
```

Comment:
This breaks my bank' files. I have no `BICFI` tag, but only `BIC`

Reply:
In my attached statement, it’s also BICFI.
In https://www.six-group.com/dam/download/banking-services/standardization/sps/ig-cash-management-sps2022-en.pdf it’s also BICFI.
https://www.iso20022.org/standardsrepository/type/BICFIIdentifier seems to say that it’s BICFI, although I’m not sure I’m reading this correctly.
https://www.iso20022.org/sites/default/files/2020-12/ISO20022_MDRPart2_PaymentsInitiation_2020_2021_v1_ForSEGReview.pdf also seems to know only BICFI.

I think your bank’s format might be wrong‽

Reply:
I included an anonymised sample file in #13 back when I wrote the patches.  
The patches are based on camt.053. I didn't look much at iso20022 itself beyond learning that camt.053 was a subset of it.

Reply:
> This breaks my bank' files. I have no `BICFI` tag, but only `BIC`

I figured out why that is: camt.053.001 up to version 02 used BIC, starting with 03 it is BICFI. At least AFAICT from the "Bank to Customer Cash Management" documents on https://www.iso20022.org/catalogue-messages/iso-20022-messages-archive
A statement specifies at the top which version it implements:
```
<Document xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.04">
```
Do yours say camt.053.001.02?

It would probably be great to support different versions and at least have a good standard mechanism to differentiate between versions.
Any idea how to do this?

Reply:
Mine says `xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.02"`, so yes, you're right, it seems like format differences. I think it would make sense to distinguish by the xml namespace. I think differences should not be too big, so we can just wrap them in if statements.

Reply:
I’m about to try and create subclasses of the Iso20022Parser to use for the different document standard versions. AFAIU that’s a good way to only have to "override" for the differences.

It seems to me that using ifs might get unwieldy some time in the future, we are already at version 11 of the standard. Let me know if you think that’s a worse approach than if statements. (I’ll start doing that now in any case because it seems like a good learning opportunity for a python noob (me).)

Meanwhile, regarding this pull request: could/would you cherry-pick the patches from RealStickman you like, that cause no trouble for your v02 documents, etc.? Then we can close this pull request and start a separate one once we have a new structure for dealing with the separate document versions.

Reply:
> I’m about to try and create subclasses of the Iso20022Parser to use for the different document standard versions

Ok, sounds good.

> could/would you cherry-pick the patches from RealStickman you like, that cause no trouble for your v02 documents, etc.?

I'm hesitating merging anything without unit tests because I can't ensure it is going to work in the long run. It is very easy to forget what that 10-years old code is doing and if there's no way to check it will stop working eventually.

So I'd rather wait for something complete. Feel free to use this PR for this work (or start new one if you find it easier)

Reply:
> Mine says `xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.02"`, so yes, you're right, it seems like format differences. I think it would make sense to distinguish by the xml namespace. I think differences should not be too big, so we can just wrap them in if statements.

Can you provide a 02 variant file for adding a test so I can test my changes to deal with different formats, please?
(See https://github.com/henk84/ofxstatement-iso20022/tree/refactor/standard_version_specific_parsers2 for how I’m trying to do this. Any comments are appreciated!)

Reply:
I believe it is a part of a test suite: https://github.com/kedder/ofxstatement-iso20022/blob/master/tests/samples/camt052.xml

Reply:
> I believe it is a part of a test suite: https://github.com/kedder/ofxstatement-iso20022/blob/master/tests/samples/camt052.xml

That’s a camt052, not camt053.
You said:
> Mine says xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.02", so yes, you're right, it seems like format differences.
and I’m assuming you copy&pasted that, so I’m also assuming that you also have a camt053 file with "minor version" 02. Is that correct?